webapp/src/components/dataset_input/TextDatasetInput.vue

```diff
@@ -10,20 +10,27 @@
 import { Set } from "immutable";
 import { ref, watch } from "vue";
 
-import type { Text } from "@epfml/discojs";
-import { Dataset } from "@epfml/discojs";
+import type { Text, Task } from "@epfml/discojs";
+import { Dataset, models } from "@epfml/discojs";
 import { loadText } from "@epfml/discojs-web";
 
 import DatasetInput from "./DatasetInput.vue";
 import FileSelection from "./FileSelection.vue";
 
+const { task } = defineProps<{ task: Task }>()
```

Comment:
I tried pretty hard to avoid referencing `Task` (the Ugly Global as I call it in my sleep), hopefully, it'll won't be needed with the latest changes to `loadText`